docs/source/libraries.rst

```diff
@@ -7,47 +7,51 @@ Here is where the Libraries API's descriptions are located:
++---------------+---------------+---------------+---------------+-------------------+
+| GPIO          | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| Hall Sensor   | Yes           | Not Supported | Not Supported |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| I2C           | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| I2S           | No            | No            | No            | WIP               |
++---------------+---------------+---------------+---------------+-------------------+
+| LEDC          | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| Motor PWM     | No            | Not Supported | Not Supported |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| Pulse Counter | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| RMT           | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| SDIO/SPI      | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| Timer         | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| Temp. Sensor  | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| Touch         | Yes           | Yes           | Not Supported |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| TWAI          | No            | No            | No            |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| UART          | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
+| USB           | Not Supported | Yes           | Yes           | ESP32-C3 only CDC |
++---------------+---------------+---------------+---------------+-------------------+
+| Wi-Fi         | Yes           | Yes           | Yes           |                   |
++---------------+---------------+---------------+---------------+-------------------+
 
 Some peripherals are not available for all ESP32 families. To see more details about it, see the corresponding datasheet.
```

Comment:
Table looks better. Here is probably missing the link to the product selector?

Reply:
Done. Tks!!